src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -278,13 +278,26 @@ public void writeBody(PrintWriter w) {
                             }
                             if (!branchFound) continue;
                             urlFound = true;
-
                             if (!(project instanceof AbstractProject && ((AbstractProject) project).isDisabled())) {
```

Comment:
isBuildable() is designed for the WebUI layout. It does not actually mean that the job is buildable (sic!)